packages/botonic-cli/src/botonic-api-service.ts

```diff
@@ -243,39 +257,55 @@ export class BotonicAPIService {
     campaign: any
   ): Promise<any> {
     const signupData = { email, password, org_name: orgName, campaign }
-    return this.apiPost({ path: '/v1/users/', body: signupData })
+    return this.apiPost({ path: 'users/', body: signupData })
   }
 
-  async saveBot(botName: string): Promise<AxiosPromise> {
+  async createBot(botName: string): Promise<AxiosPromise> {
     const resp = await this.apiPost({
-      path: '/v2/bots/',
+      apiVersion: 'v2',
+      path: 'bots/',
       body: { name: botName },
     })
-    if (resp.data) this.setCurrentBot(resp.data)
+
+    if (resp.data) {
+      this.setCurrentBot(resp.data)
+    }
+
     return resp
   }
 
-  async getMe(): Promise<AxiosPromise> {
-    return this.apiGet({ path: '/v1/users/me/' })
+  private async getMe(): AxiosPromise<Me> {
+    return this.apiGet({ path: 'users/me/' })
   }
 
   async getBots(): AxiosPromise<BotsList> {
-    return this.apiGet({ path: '/v2/bots/' })
+    const botsResponse = await this.apiGet({ apiVersion: 'v2', path: 'bots/' })
+
+    if (botsResponse.data.next) {
+      this.getMoreBots(botsResponse.data.results, botsResponse.data.next)
+    }
+
+    return botsResponse
   }
 
-  async getMoreBots(bots: any, nextBots: any) {
-    if (!nextBots) return bots
+  private async getMoreBots(bots: BotListItem[], nextBots?: string) {
+    if (!nextBots) {
+      return bots
+    }
+
     const resp = await this.apiGet({
-      path: nextBots.split(this.baseUrl)[1],
+      apiVersion: 'v2',
+      path: nextBots.split(`${this.baseUrl}/v2/`)[1],
     })
     resp.data.results.map(b => bots.push(b))
     nextBots = resp.data.next
+
     return this.getMoreBots(bots, nextBots)
   }
 
   async getProviders(): Promise<AxiosPromise> {
     return this.apiGet({
-      path: '/v1/provider_accounts/',
+      path: 'provider_accounts/',
```

Comment:
i'm wondering if it will be helpful to have a kind of URL builder or helper. I think that this trailing slash `/` sometimes makes the request to work or not, so having something to avoid this kind of problems would be nice. Up to you

Reply:
This seems to be because of the configuration of django in the backend